File: konwerter temperatur.py
def konwerter_temperatur():
    try:
        kierunek = input("Wybierz konwersję (C - Celsjusz na Fahrenheit, F - Fahrenheit na Celsjusz): ").upper()
        temperatura = float(input("Podaj temperaturę: "))

        if kierunek == 'C':
            wynik = temperatura * 1.8 + 32
            print(f"{temperatura}°C = {wynik}°F")
        elif kierunek == 'F':
            wynik = (temperatura - 32) / 1.8
            print(f"{temperatura}°F = {wynik:.2f}°C")
        else:
            print("Niepoprawny wybór konwersji.")
            print("Program zakończony.")
            exit()
    finally:
        print("Program zakończony.")
        exit()

konwerter_temperatur()

File: konwerter temperatur.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        Console.Write("Wybierz konwersję (C - Celsjusz → Fahrenheit, F - Fahrenheit → Celsjusz): ");
        string direction = Console.ReadLine().ToUpper();

        Console.Write("Podaj temperaturę: ");
        double inputTemp = Convert.ToDouble(Console.ReadLine());

        double converted;

        if (direction == "C")
        {
            converted = inputTemp * 1.8 + 32;
            Console.WriteLine($"{inputTemp}°C = {converted}°F");
        }
        else if (direction == "F")
        {
            converted = (inputTemp - 32) / 1.8;
            Console.WriteLine($"{inputTemp}°F = {converted}°C");
        }
        else
        {
            Console.WriteLine("Nieznany kierunek konwersji.");
            return;
        }

    }
}
